test/generators/typescript/TypeScriptGenerator.spec.ts

```diff
@@ -128,8 +133,10 @@ ${content}`;
     };
 
     generator = new TypeScriptGenerator({ modelType: 'interface' });
-    const models = await generator.generate(doc);
-    expect(models).toHaveLength(1);
+    const models = await generator.generateCompleteModels(doc, {
+      exportType: 'named'
+    });
+    expect(models).toHaveLength(2);
```

Comment:
If you want to test the generated complete models, please create a test just for this 🙂 As it's two different renderings and what they test :v: